storage/oasis/nodeapi/universal_runtime.go

```diff
@@ -135,8 +136,22 @@ func (rc *UniversalRuntimeApiLite) GetEventsRaw(ctx context.Context, round uint6
 	return evs, nil
 }
 
-func (rc *UniversalRuntimeApiLite) EVMSimulateCall(ctx context.Context, round uint64, gasPrice []byte, gasLimit uint64, caller []byte, address []byte, value []byte, data []byte) ([]byte, error) {
-	return rc.sdkClient.Evm.SimulateCall(ctx, round, gasPrice, gasLimit, caller, address, value, data)
+func (rc *UniversalRuntimeApiLite) EVMSimulateCall(ctx context.Context, round uint64, gasPrice []byte, gasLimit uint64, caller []byte, address []byte, value []byte, data []byte) (*FallibleResponse[[]byte], error) {
+	res, err := rc.sdkClient.Evm.SimulateCall(ctx, round, gasPrice, gasLimit, caller, address, value, data)
+	if errors.Is(err, ErrSdkEVMExecutionFailed) || errors.Is(err, ErrSdkEVMReverted) {
+		localErr := sdkToLocalErr[err]
+		return &FallibleResponse[[]byte]{
+			Result:           nil,
```

Comment:
Nit here and below: I think it's more idiomatic to omit the initialization to `nil`. Feel free to leave as-is if you find this more readable.